src/site/content/en/blog/native-lazy-loading/index.md

```diff
@@ -0,0 +1,304 @@
+  on Chrome for Android
+- The [effective connection type](https://googlechrome.github.io/samples/network-information/)
+
+You can find the default values for the different effective connection types in the [Chromium
+source](https://cs.chromium.org/chromium/src/third_party/blink/renderer/core/frame/settings.json5?l=971-1003&rcl=e8f3cf0bbe085fee0d1b468e84395aad3ebb2cad).
+These numbers, and even the approach of fetching only when a certain distance from the viewport is
+reached, may change in the near future as the Chrome team improves heuristics to determine when to
+begin loading.
+
+{% Aside %}
+There is an [open bug](https://bugs.chromium.org/p/chromium/issues/detail?id=982020&q=component:Blink) to ensure that these different thresholds can be experimented with by [throttling the network](https://developers.google.com/web/tools/chrome-devtools/network/#throttle) in Chrome DevTools. In the meantime, you will need to override the effective connection type of the browser using the `chrome://flags/#force-effective-connection-type` flag.
+{% endAside %}
+
+### Image loading
+
+To prevent the surrounding content from reflowing when a lazy loaded image is downloaded, make sure
+to add `height` and `width` attributes to the `<img>` element or specify their values directly in an
+inline style:
+
+```html
+<img src="..." loading="lazy" width="200" height="200">
+<img src="..." loading="lazy" style="height:200px; width:200px;">
+```
+
+Images will still load lazily if dimensions are not included, but specifying them decreases the chance of
+browser reflow.
+
+Support for the `intrinsicsize` attribute is also being [worked
+on](https://bugs.chromium.org/p/chromium/issues/detail?id=967992), so images will lazy load
+correctly if `intrinsicsize` is specified along with one other dimension (`width` or `height`).
+
+```html
+<img src="…" alt="…" loading="lazy" intrinsicsize="250x200" width="450">
+<!-- lazy loaded -->
+```
+
+{% Aside %}
+  Take a look at this [demo](https://mathiasbynens.be/demo/img-loading-lazy) to see how the `loading` attribute works with 100 pictures.
+{% endAside %}
+
+### iframe loading
+
+The `loading` attribute affects iframes differently than images, depending on whether the iframe is
+hidden. (Hidden iframes are often used for analytics or communication purposes.) Chrome uses the
+following criteria to determine whether an iframe is hidden:
+
+- The iframe's width and height are 4 px or smaller.
+- `display: none` or `visibility: hidden` is applied.
+- The iframe is placed off-screen using negative X or Y positioning.
+
+If an iframe meets any of these conditions, Chrome considers it hidden and won't lazy load it in
+most cases. Iframes that _aren't_ hidden will only load when they're within the [load-in distance threshold](#load-in-distance-threshold).
+A placeholder shows for lazily loaded iframes that are still being fetched.
+
+### Feature Policy
```

Comment:
Has anyone verified that the feature policies work?

Reply:
Was under the impression that they're still being worked on (don't see the policy for images with `document.featurePolicy.features()` even with `--enable-experimental-web-platform-features` turned on)

Will definitely like to test them both and even include an example of them into the article (not blocking and something I can add later)